ci/taos/plugins-staging/pr-nnstreamer-ubuntu-apptest.sh

```diff
@@ -136,7 +144,9 @@ function pr-nnstreamer-ubuntu-apptest-run-queue() {
         # Test that video mixer with nnstreamer plug-in
         # Testing while 2seconds. 2seconds is arbitrary.
         # and then kill process, otherwise, process run forever.
-        ./nnstreamer_example_cam &
+        echo -e "" >> ../../report/pr-nnstreamer-apptest.log
+        echo -e "[DEBUG] Starting nnstreamer_example_cam test..." >> ../../report/pr-nnstreamer-apptest.log
+        ./nnstreamer_example_cam &>> ../../report/pr-nnstreamer-apptest.log &
```

Comment:
Please enable fake USB camera after this PR. 
The solution can be found at https://github.com/nnsuite/nnstreamer/issues/531.